data/is_ocaml_yet/web.md

```diff
@@ -0,0 +1,258 @@
+---
+id: web
```

Comment:
As the question of the state of the art of static blog generators in OCaml is raised quite often on the forum, I think adding a section dedicated to them is a good thing.

Reply:
Great idea! I added a section with a few packages